workshop-prereqs.adoc

```diff
@@ -0,0 +1,33 @@
+:toc:
+
+= Prereqs
+
+Here are some prereqs for link:readme.adoc[Kubernetes on AWS workshop]
+
+- You will be using an AWS Account while you go through the workshop. link:http://docs.aws.amazon.com/AmazonSimpleDB/latest/DeveloperGuide/AboutAWSAccounts.html[Create an AWS Account] if you don't already have one
+- If you already have an AWS Account, you need to create an IAM user and attach these http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html[IAM policies].
+
+    AmazonEC2FullAccess
+    AmazonRoute53FullAccess
+    AmazonS3FullAccess
+    IAMFullAccess
+    AmazonVPCFullAccess
+
+Please review this link:https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user[link]
+for additional info on IAM permissions
+
+- Install the latest version of http://docs.aws.amazon.com/cli/latest/userguide/installing.html[AWS CLI]
+on your PC
+- Use {TBD} AWS region for this workshop. We have done minimal testing against these AWS regions.
+For ex, installing and terminating k8s cluster. Please file a bug if you run into issues
+
+    US East (N. Virginia) us-east-1
+    US West (Oregon)      us-west-2
+    EU (Frankfurt)        eu-central-1
+    EU (Ireland)          eu-west-1
+
+You can set the default region
+
+    export AWS_DEFAULT_REGION=us-central-1
```

Comment:
Can we change this to the default region that will be used for RI?

Reply:
I'm guessing we will use Frankfurt/Ireland for both Container summit and reInvent. I think we should test against these regions and choose one. I will advocate folks to use these regions in chat room and make the change once we have consensus